charts/vsphere-csi/v2.1.0/questions.yaml

```diff
@@ -10,36 +10,36 @@ questions:
     group: Configuration
     show_subquestion_if: true
     subquestions:
-      - variable: vCenter.host
+      - variable: csiController.vsphereConfig.host
```

Comment:
This should be consistent with the vSphere CPI chart, we should place the vSphere config stuff under `cloudConfig.` instead of `csiController.vsphereConfig.` or modify the other PR accordingly

Reply:
The cloudConfig is specific to CPI because it creates a config map. CSI in the other hand has a `csi-vsphere.conf`, which is what's contained within the secret.